### ISSUE TITLE
Support to view public buckets

### ISSUE THREAD

**Issue by muety** [cloudlena/s3manager#27]
I'd love to be able to use this tool to browse "public" buckets outside of my account, e.g. `s3://sentinel-s2-l1c`.

**Comment by nkicg6** [comment]
Agreed this would be useful! In the meantime, if you enter the bucket name directly into the url you can explore it as is:

`http://localhost:8080/buckets/<public-bucket-name-here>`
For example, `http://localhost:8080/buckets/dataworld-linked-acs`

Make sure you set `LIST_RECURSIVE=true`. You still need to enter your AWS credentials, but I found this works for some public buckets I've tested.
